packages/plugin-layout/src/ZoneNode.ts

```diff
@@ -0,0 +1,36 @@
+import { VNode } from '../../core/src/VNodes/VNode';
+
+export type ZoneIdentifier = string;
+
+export class ZoneNode extends VNode {
+    hidden: Map<VNode, boolean> = new Map();
+
+    constructor(public managedZones: ZoneIdentifier[]) {
```

Comment:
Why one zone can manages multiples zone?
I could see one zone needing to manage multiples components but no other nodes.

Reply:
dialog who manage float, default for example.